### PR TITLE
[GOV] proposed lazy consensus policy

### PR DESCRIPTION
**Pull request by fkiraly**
This document updates the lazy consensus policy to the de-facto rules.

How the de-facto rules came into being:

* with pydata conferences in Berlin and London approaching in Apr, a lot of PR were stalled due to lack of review
* CC then accepted a standing policy of letting PR by core devs to be merged after 5 WD without review, and 3 WD in the week before the conference was 
* this was enacted twice, since there were two conferences (Berlin, London)
* afterwards it was agreed to keep the 5 WD rule as a standing policy, to prevent PR stalling

in summary, we have been working like this since the lead-up to pydata Berlin.
@MatthewMiddlehurst therefore made the valid point that we need to adapt the governance document and perhaps have a discussion of what the rule should be.

This PR changes the rules to the current "status quo", i.e., moving the standing policy into the gov docs.

### CODE REVIEW

**Comment by miraep8** [comment]
What happens if no one reviews this PR in 5 days??????

image

(Just in case of mis-interpretation, meme very meant as a light-hearted joke about the PR, not a personal attack)

**Comment by fkiraly** [comment]
> What happens if no one reviews this PR in 5 days??????

Yeah, self-recursion...

The standing policy did only affect code (from a precedent viewpoint), so I think this still needs at least one approval.
Also given that people might find it controversial, even if we were to apply the standing policy to this, we probably should not.

But, this highlights another problem with changing the policy: the danger of "stealth modifications" to the governance document - or, core interfaces!

I'll update it so that only changes that do not require a STEP can be moved in via lazier consensus.

File: docs/source/get_involved/governance.rst [hunk 1]
```diff
@@ -400,9 +400,11 @@ core developers.
    Ten working days excluding week-ends constitute "reasonable time" in the above sense.
    The period resets at every new change made to the PR.
    It starts only when all GitHub checks pass.
```
Comment (MatthewMiddlehurst) [inline docs/source/get_involved/governance.rst:402]: Just to clarify, the "reasonable time" period is 10 days for non-core devs, and 5 days for core devs (as per the below rule change)?

File: docs/source/get_involved/governance.rst [hunk 2]
```diff
@@ -400,9 +400,11 @@ core developers.
...
--  During this period, the PR can be merged if it has an approval and no rejection, but should be
-   reverted if it receives a rejection in addition.
--  If the "reasonable time" period elapses and no approval or rejection has been expressed on a PR,
+-  During this period, the PR can be merged if it has an approval and no rejection after two working days excluding week-ends,
+   or if five working days excluding week-ends have passed (without any additional condition).
```
Comment (MatthewMiddlehurst) [inline docs/source/get_involved/governance.rst:404]: Perhaps "without any additional condition" is a bit strong, maybe a rejection should halt this 🙂. That would be the case currently, correct?

Comment (fkiraly) [reply docs/source/get_involved/governance.rst]: that's not what I meant, I meant if there is no review activity, and it's authored by a core dev, it can be merged.
